dev/io.openliberty.session.6.1.internal/bnd.bnd

```diff
@@ -0,0 +1,46 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: WAS Session
+Bundle-SymbolicName: io.openliberty.session.6.1.internal
+Bundle-Description: WAS Session for Servlet 6.1
+
+javac.source: 17
+javac.target: 17
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
+
+# For each exported package, create (in that package) a package-info.java
+# file, and place an @version javadoc tag in package-level javadoc. 
+# Append ";provide:=true" if this bundle also provides an implementation
+# for the exported API.
+Export-Package: \
+    io.openliberty.session;provide:=true, \
+    io.openliberty.session.http;provide:=true, \
+    io.openliberty.session.resources;provide:=true
```

Comment:
Usually we do not export the nls resources.  Are you do so for a reason?

Reply:
Initially, I wanted to reused 6.0 nls and add new message for 6.1 but it does not come out. ...(maybe because of the Internal-Package?)

Reply:
I want to centralize the message ID into 6.1 from now on.  i.e If there is new messages in 7.0 release, they will be put into this 6.1 resource file.

Reply:
Yeah you could use the 6.0 one.  You would need to rename it to remove the `.internal.` from the package name of the 6.0 resource bundle and add an import to the from the 6.1 bundle's bnd.bnd file.